conf/radiusd/eduroam.conf.example

```diff
@@ -0,0 +1,132 @@
+
+pidfile = %%pid_file%%
+
+$INCLUDE radiusd.conf
+
+listen {
+    ipaddr = %%management_ip%%
+    port = %%eduroam_auth_listening_port%%
+    type = auth
+    virtual_server = eduroam
+}
+
+listen {
+    type = control
+    socket = %%socket_file%%
+    mode = rw
+}
+
+log {
+	#
+	#  Destination for log messages.  This can be one of:
+	#
+	#	files - log to "file", as defined below.
+	#	syslog - to syslog (see also the "syslog_facility", below.
+	#	stdout - standard output
+	#	stderr - standard error.
+	#
+	#  The command-line option "-X" over-rides this option, and forces
+	#  logging to go to stdout.
+	#
+	destination = files
+
+	#
+	#  Highlight important messages sent to stderr and stdout.
+	#
+	#  Option will be ignored (disabled) if output if TERM is not
+	#  an xterm or output is not to a TTY.
+	#
+	colourise = yes
```

Comment:
Yeah, I tend to conserve what others do and that I don't need to change ;) :P
https://github.com/inverse-inc/packetfence/blame/devel/conf/radiusd/auth.conf.example#L46
https://github.com/inverse-inc/packetfence/blame/devel/conf/radiusd/acct.conf.example#L39